README.md

```diff
@@ -91,12 +96,19 @@ matches the constraints from the manifest. If the dependency is missing from
 
 ### Adding a dependency
 
-1. `import` the package in your `*.go` source code file(s).
-1. Run the following command to update your `Gopkg.lock` and populate `vendor/` with the new dependency.
+Adding a project as a dependency has three essential steps:
 
-    ```sh
-    $ dep ensure
-    ```
+1. `import` a package from the project in one of your `*.go` source files
+2. Add a version constraint on the project to `Gopkg.toml` (Optional, but recommended)
+3. Run `dep ensure`
+
+`dep ensure -add` provides some CLI sugar to ease this process:
```

Comment:
Now that `-add` is a fully operational battlestar, my comments are all out-of-date. 😀   Thanks for pushing that through!!!

Since this is the quickstart/readme, how about telling people just the best practice:

1. Run `dep ensure -add github.com/foo/bar`. This adds a constraint for the latest version to your Gopkg.toml, then updates the lock and vendor directory.
2. Import and use the package in your code. ✨ 

Sparkles optional, but highly recommended.